### PR TITLE
fix lint errors for lablqt related packages

### PR DESCRIPTION
**Pull request by Kakadu**
Kind of inspired by https://github.com/ocaml/opam-repository/pull/5652

@dsheets

### CODE REVIEW

**Comment by dsheets** [comment]
Great! Thanks!

**Comment by dsheets** [comment]
It looks like the findlib package `lablqml` is attempted to be installed multiple times.

**Comment by Kakadu** [comment]
@dsheets, checks have passed

File: packages/lablqt/lablqt.0.2/opam [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
+maintainer:   "kakadu.hafanana@gmail.com"
+authors:      "kakadu.hafanana@gmail.com"
+homepage:     "http://kakadu.github.io/lablqt/"
+bug-reports:  "https://github.com/kakadu/lablqt/issues"
+dev-repo:     "https://github.com/Kakadu/lablqt.git"
+
 build: [
   ["./configure"]
   [make "generator"]
   ["bash" "-c" "cd lablqml && ./configure --datarootdir=%{lib}% && %{make}%" ]
```
Comment (dsheets) [inline packages/lablqt/lablqt.0.2/opam]: Still need `bash`?

File: packages/lablqt/lablqt.0.3/opam [hunk 2]
```diff
@@ -1,9 +1,17 @@
...
+homepage:     "http://kakadu.github.io/lablqt/"
+bug-reports:  "https://github.com/kakadu/lablqt/issues"
+dev-repo:     "https://github.com/Kakadu/lablqt.git"
+
+available:    [ocaml-version >= "4.02.0"]
 build: [
   ["./configure"]
   [make "generator"]
-  ["bash" "-c" "cd lablqml && ./configure --datarootdir=%{lib}% && %{make}%" ]
+  ["bash" "-exc" "cd lablqml && ./configure --datarootdir=%{lib}% && %{make}%" ]
```
Comment (hannesm) [inline packages/lablqt/lablqt.0.3/opam]: there's still some callout to bash here...

**Comment by avsm** [comment]
This is an improvement, so merging.

**Pull request merged**